deltachat-jsonrpc/src/api/types/message.rs

```diff
@@ -130,6 +130,11 @@ impl MessageObject {
 
         let webxdc_info = if message.get_viewtype() == Viewtype::Webxdc {
             Some(WebxdcMessageInfo::get_for_message(context, msg_id).await?)
+        } else if let (deltachat::mimeparser::SystemMessage::WebxdcInfoMessage, Some(parent_msg)) =
+            (message.get_info_type(), message.parent(context).await?)
```

Comment:
Oh, i didn't noticed in the first version of the PR that the parent message is also used below:
```Rust
let parent_id = message.parent(context).await?.map(|m| m.get_id().to_u32());
```
so it should be loaded in advance, before this `let` statement.

Reply:
But is that not how it was implemented initially?

Reply:
True, only `unwrap()` wasn't needed because it's better to use the `if let` pattern matching